app/controllers/translations_controller.rb

```diff
@@ -1,12 +1,24 @@
 # frozen_string_literal: true
 
 class TranslationsController < ApplicationController
+  def index
+    render json: all_translations, include: params[:include], status: :ok
+  end
+
   def show
     redirect
   end
 
   private
 
+  def all_translations
+    if params['filter']
+      Translation.is_published(params['filter']['is_published'])
```

Comment:
I'm guessing you based this off `Resource.system_name.`  You'll need to create a scope in `Translation` if you want this to work.  However, I don't think we want to return drafts here, since this is an unsecured endpoint.  This should probably only return drafts